docs/101/allocations.md

```diff
@@ -26,23 +26,73 @@ accounts balance
 ```
 
 ## How do I review the usage of my allocation?
-The `accounts` tool provides several options for summarizing allocation usage. To obtain a summary, execute the command:
+The `accounts` tool provides several options for summarizing the usage of the allocations to which your RCC account is accessible. To obtain a summary, execute the command `accounts usage` with the account name(s):
+
+
+=== "Midway2, DaLI"
+      ```
+      accounts usage
+      accounts usage --account [pi-cnetid]
+      accounts usage --accounts [pi-cnetid1],[pi-cnetid2]
+      ```
```

Comment:
all work as shown.

something related: 
- help shows "-account -a ACCOUNT[,...]", i.e., the option "account" with single dash, i.e., "-", and doesn't show any option of "accounts". 
- "-a" as shown in help doesn't work.
- the example of "accounts" above will work just fine with "account", and otherwise too.

Reply:
Here it looks to me on midway2
```
[ndtrung@midway2-login1 ~]$ accounts usage --accounts pi-depablo,pi-gagalli
Account Usage: Thu Aug 29 15:32:56 2024
Accounts:   pi-depablo,pi-gagalli

+------------+-------------+
| Account    | Charge (SU) |
+------------+-------------+
| pi-depablo |  2781855.77 |
| pi-gagalli |  1337831.94 |
+------------+-------------+
```